skipchain/skipchain_test.go

```diff
@@ -113,7 +113,7 @@ func storeSkipBlock(t *testing.T, nbrServers int, fail bool) {
 
 	// And add it again, with all nodes running
 	if fail {
-		log.Lvl3("Unpausing server ", deadServer.Address())
+		log.LLvl3("Unpausing server ", deadServer.Address())
```

Comment:
Checked in an LL, oops.

Reply:
yes, still WIP because it doesn't work yet - I thought (again) that this was a low hanging fruit :(